File: Iniciante/URI 1017.c
#include <stdio.h>
 
int main() { 
    double kilometros, media;
    int tempo, velocidade;
	
    scanf("%d %d", &tempo, &velocidade);
    kilometros = tempo * velocidade;
	media = kilometros / 12;
    printf("%.3lf\n", media);

    return 0;
}

File: Iniciante/URI 1017.go
package main
 
import (
    "fmt"
)
 
func main() {
    var tempo, velocidade, kilometros, media float64

    fmt.Scanf("%f", &tempo)
    fmt.Scanf("%f", &velocidade)
    kilometros = tempo * velocidade
    media = kilometros / 12
    fmt.Printf("%.3f\n", media)
}